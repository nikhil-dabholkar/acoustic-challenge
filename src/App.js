import React from 'react';
import './App.css';
import 'bootstrap/dist/css/bootstrap.min.css';
import 'ag-grid-community/dist/styles/ag-grid.css';
import 'ag-grid-community/dist/styles/ag-theme-alpine.css';
import RootContainer from './containers/RootContainer/RootContainer';

function App() {
  return (
    <div className="App">
      <RootContainer></RootContainer>
    </div>
  );
}

export default App;
