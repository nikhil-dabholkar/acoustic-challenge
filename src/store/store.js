import rootReducer from '../reducer/index';
import thunk from 'redux-thunk';
import { createStore, applyMiddleware } from 'redux';

export const store = createStore(rootReducer, applyMiddleware(thunk))