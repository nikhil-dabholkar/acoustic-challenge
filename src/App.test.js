import React from 'react';
import App from './App';
import {IntlProvider} from 'react-intl';
import { store } from "./store/store";
import { Provider } from 'react-redux';
import { render, cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";
import renderer from "react-test-renderer";

afterEach(cleanup);

test('renders learn react link', () => {
  const { getByText } = render(<Provider store={store}><IntlProvider locale="en"><App></App></IntlProvider></Provider>);

});

it("matches snapshot", () => {
    const tree = renderer.create(<Provider store={store}><IntlProvider locale="en"><App></App></IntlProvider></Provider>).toJSON();
    expect(tree).toMatchSnapshot();
})
