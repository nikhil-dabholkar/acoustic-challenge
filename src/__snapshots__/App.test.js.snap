// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="App"
>
  <div
    data-testid="rootContainer"
  >
    <div
      data-testid="headerContainer"
    />
    <div
      data-testid="headerComponent"
    >
      <nav
        className="navbar navbar-light bg-white"
      >
        <a
          className="navbar-brand"
          href="/"
        >
          <img
            alt=""
            className="d-inline-block align-top"
            src="logo.png"
            width="100px"
          />
        </a>
        <span
          className="languageSwitcherContainer"
        >
          <div
            data-testid="languageSwitcherComponent"
          >
            <span>
              English
               | 
            </span>
            <span>
              <a
                className="bx--link"
                href="#"
                onClick={[Function]}
              >
                Polish
              </a>
              
            </span>
          </div>
        </span>
      </nav>
      <nav
        className="navbar navbar-expand-lg navbar-light bg-light"
      >
        <div
          className="collapse navbar-collapse"
          id="navbarNav"
        >
          <ul
            className="navbar-nav"
          >
            <li
              className="nav-item active"
            >
              <a
                className="nav-link"
                href="/"
                onClick={[Function]}
              >
                Home
              </a>
            </li>
            <li
              className="nav-item"
            >
              <a
                className="nav-link"
                href="/articles"
                onClick={[Function]}
              >
                Articles
              </a>
            </li>
          </ul>
        </div>
      </nav>
    </div>
    <div
      className="jumbotron jumbotron-fluid"
    >
      <div
        data-testid="homeContainer"
      />
      <img
        src="https://i2.wp.com/spyderslab.com/wp-content/uploads/2018/06/digital_marketing_spyderslab.jpg?resize=1170%2C480&ssl=1"
      />
    </div>
  </div>
</div>
`;
